packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1512,7 +1512,7 @@ The following response is only a subset of properties for brevity.
 
 Fetches the schema for a Log Stream type. The `${typeId}` element in the URL specifies the Log Stream type, which is either `aws_eventbridge` or `splunk_cloud_logstreaming`. Use the `aws_eventbridge` literal to retrieve the AWS EventBridge type schema, and use the `splunk_cloud_logstreaming` literal retrieve the Splunk Cloud type schema.
 
-See [Log Streaming API](/docs/reference/api/log-streaming) for examples of Log Stream objects.
+See [Log Streaming API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/LogStream/) for examples of Log Stream objects.
```

Comment:
Are we going to be adding separate OpenAPI docs for schemas too?

Reply:
Yes, the Schemas API is separate: https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Schema/ (it's not complete and will need lots of work to get it up-to-date)